URBNCreativeTestRig/FontsTableViewController.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  FontsTableViewController.swift
+//  URBNCreativeTestRig
+//
+//  Created by Jason Grandelli on 11/12/15.
+//  Copyright © 2015 Jason Grandelli. All rights reserved.
+//
+
+import UIKit
+import URBNDataSource
+
+class FontsTableViewController: UITableViewController, UISearchResultsUpdating {
+
+    private let completion: (fontName: String) -> (Void)
+    private let dataSource = URBNArrayDataSourceAdapter(items: nil)
+    private var tableData: Array<String> = []
+    private lazy var typeFaces: Array<String> = {
+        var fonts: Array<String> = []
```

Comment:
idk, that was my original stance, but everyone else writing swift was going with the swift syntax so I just went with it. now I'm used to it